src/screens/hardware-wallets/PairHardwareWalletSigningSheet.tsx

```diff
@@ -168,13 +168,28 @@ export function PairHardwareWalletSigningSheet() {
           },
         });
       } else {
-        Alert.alert(
-          'Error',
-          'Something went wrong. Please restart the app and try again.'
+        logger.error(
+          new RainbowError(
+            '[importHardwareWallet] - Disconnected or Unkown Error'
+          ),
+          { errorType }
         );
+        logger.info('[importHardwareWallet] - issue connecting, trying again ');
+        const transport = await TransportBLE.open(deviceId);
+        await checkLedgerConnection({
+          transport,
+          deviceId,
+          successCallback,
+          errorCallback: () => {
+            Alert.alert(
+              i18n.t(TRANSLATIONS.pairing_error_alert.title),
+              i18n.t(TRANSLATIONS.pairing_error_alert.body)
+            );
+          },
```

Comment:
Is this just checking _why_ something happened after it errors out?

Reply:
when a user opens or closes the eth app on the device the connection will close and become disconnected.in this instance it should be the user opening the app, this handles that scenario and attempts to reconnect